lib/data_state.hpp

```diff
@@ -13,9 +13,15 @@
 #define GLVIS_DATA_STATE_HPP
 
 #include <string>
-#include <vector>
 #include <memory>
-#include "mfem.hpp"
+
+#include <mfem.hpp>
+using mfem::Mesh;
+using mfem::Vector;
+using mfem::GridFunction;
+using mfem::QuadratureFunction;
+using mfem::DataCollection;
```

Comment:
Why to do this? It would be fine in a cpp file, but a header file it is "contagious", spreading in an uncontrolled way 🫤 . Maybe scope it by putting it to `DataState`? 😉

Reply:
Trying to avoid the 'using namespace' in the header and not poluting with too many 'mfem::'. 
Good idea moving it inside the scope.

Reply:
Scoping was not working, I cleaned up the header files by using `mfem::` specifically.